Header header
float64[] pos
float64[] pos_sigma
float64[] vel
float64[] vel_sigma
uint32    num_svs